bugtool/cmd/configuration.go

```diff
@@ -98,6 +98,12 @@ func defaultCommands(confDir string, cmdDir string, k8sPods []string) []string {
 		"ip6tables -S",
 		"iptables -L -v",
 		"ip rule",
+		"iptables-legacy -L -v",
+		"iptables-legacy -S",
+		"ip6tables-legacy -S",
+		"iptables-nft -L -v",
+		"iptables-nft -S",
+		"ip6tables-nft -S",
```

Comment:
Do we also need `iptables-legacy-save -c` and `iptables-nft-save -c`?

Reply:
For what it's worth, I personally avoid `iptables` in favour of `iptables-save` because I know that `iptables` commands by themselves tend to omit some tables or chains, whereas I know that iptables-save won't lie to me. My guess would be that this criticism is equally applicable to the -nft / -legacy approaches since upstream probably try to retain compatibility in the output format between the releases.

Reply:
> I am not sure if its really needed .  I tried dumping them locally and `iptables-legacy-save -c` and ` iptables-legacy -c`  and they both were identical 

I don't see a `iptables-legacy -c`. Did you mean something else?
```
# iptables-legacy -V
iptables v1.8.4 (legacy)
# iptables-legacy -c
iptables v1.8.4 (legacy): option "-c" requires an argument
Try `iptables -h' or 'iptables --help' for more information.
```
<br>

Also agree with Joe. Plus, I find `iptables-save` has better context on each line, which helps when grepping.

Reply:
@pchaigno sorry I made a typing error , I meant to say `iptables-save -c` . Also i got joe's point , so shall i remove `iptables` command from bugtool and instead use `iptables-save` ?

Reply:
I would keep both to be consistent with the rest.